Gemfile

```diff
@@ -21,6 +21,7 @@ group :development, :test do
   gem 'binding_of_caller'
   gem 'grape_on_rails_routes'
   gem 'haml_lint'
+  gem 'rails_real_favicon'
```

Comment:
I think we don't need to keep this gem here, since it's only necessary when adding the favicons first time. So keeping it here will just add yet another gem to download.

Besides that, all good :-)